File: src/main/java/rbq/wtf/lycoris/client/transformer/transformers/BackTransformer.java
package rbq.wtf.lycoris.client.transformer.transformers;

import rbq.wtf.lycoris.client.transformer.ClassTransformer;

public class BackTransformer extends ClassTransformer {
    private final Class<?> targetClass;
    private final byte[] bytes;

    public BackTransformer(Class<?> targetClass, byte[] bytes) {
        this.targetClass = targetClass;
        this.bytes = bytes;
    }

    @Override
    public Class<?> getTargetClass() {
        return targetClass;
    }

    @Override
    public byte[] transform(byte[] bytes) {
        return this.bytes;
    }
}

File: src/main/java/rbq/wtf/lycoris/client/transformer/transformers/BackTransformer.kt
package rbq.wtf.lycoris.client.transformer.transformers

import rbq.wtf.lycoris.client.transformer.ClassTransformer

class BackTransformer(override val targetClass: Class<*>, private val bytes: ByteArray) : ClassTransformer() {
    override fun transform(bytes: ByteArray): ByteArray {
        return this.bytes
    }
}